src/Microsoft.PowerShell.Security/security/CertificateProvider.cs

```diff
@@ -31,20 +31,18 @@ namespace Microsoft.PowerShell.Commands
 {
```

Comment:
@iSazonov, your last commit had 4 failures in `PowerShell-CI-windows`
Invoke-WebRequest tests.Denial of service.Charset Parsing
```powershell
Expected the actual value to be greater than 10, but got 5.57108543031811.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Utility\WebCmdlets.Tests.ps1: line 1925
1925:             $pathologicalRatio | Should -BeGreaterThan 10
```
CmsMessage cmdlets and Get-PfxCertificate basic tests.Verify CMS message recipient resolution by path
```powershell
You cannot call a method on a null-valued expression.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Security\CmsMessage.Tests.ps1: line 57
57:         $recipient.Resolve($ExecutionContext.SessionState, "Encryption", [ref] $errors)

```
CmsMessage cmdlets and Get-PfxCertificate basic tests.Verify CMS message recipient resolution by cert
```powershell
Cannot bind argument to parameter 'FilePath' because it is null.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Security\CmsMessage.Tests.ps1: line 65
65:         $cert = Get-PfxCertificate $certLocation

```
CmsMessage cmdlets and Get-PfxCertificate basic tests.Verify a CMS message can be protected / unprotected
```powershell
Cannot bind argument to parameter 'To' because it is null.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Security\CmsMessage.Tests.ps1: line 74
74:         $protected = "Hello World","How are you?" | Protect-CmsMessage -To $certLocation

```